pkg/cache/types/types.go

```diff
@@ -41,8 +41,8 @@ type ResponseType int
 const (
```

Comment:
Let's add a big fat comment to tell people that the ordering of these enum values matters, linking to the ADS reference if possible.